tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -79,9 +79,14 @@ popd
 
 info "Install fd to destdir"
 install_dir="${DESTDIR}/${PREFIX}/share/ovmf"
-if [ "${ovmf_build}" == "tdx" ]; then
-	install_dir="$DESTDIR/$PREFIX/share/tdvf"
-fi
+case "${ovmf_build}" in
+	"tdx")
+		install_dir="$DESTDIR/$PREFIX/share/tdvf"
+		;;
+	"x86_64")
+		install_dir="$DESTDIR/$PREFIX/share/ovmfx86_64"
```

Comment:
I would prefer to have the two versions of ovmf be called `AMDSEV.fd` for the AmdSevPkg and `OVMF.fd` for the normal x64 package. This would match a bit better with the conventions that I have seen and there would be no need for this additional `ovmfx86_64` directory.

Reply:
I agree with @fitzthum. That would make the most sense here.